modules/post/linux/gather/checkvm.rb

```diff
@@ -140,19 +139,37 @@ def run
       when /vmware virtual ide|vmware pvscsi|vmware virtual platform/i
         vm = "VMware"
       when /xen_mem|xen-vbd/i
-        vm =  "Xen"
+        vm = "Xen"
       when /qemu virtual cpu version/i
         vm = "Qemu/KVM"
       when /\/dev\/vmnet/
         vm = "VMware"
       end
     end
 
+    # Check for .dockerenv file
+    if not vm
```

Comment:
I thought about this after looking at the rubocop results, but the [guidance I saw in the wiki](https://github.com/rapid7/metasploit-framework/wiki/Using-Rubocop#conditional-statements) really resonated with me (probably because I'm not a ruby expert). Also, the same `if not vm` is used throughout this file. This doesn't make `if not vm` correct, but I don't want to be the guy to mess up consistency. I have no problem changing each occurrence in this file to an `unless` if needed.